eman2/__init__.py

```diff
@@ -115,6 +115,12 @@ def createEmanProcess(cls, script='e2converter.py', args=None, direc="."):
 
         return proc
 
+    @classmethod
+    def getTemplateCommand(cls):
```

Comment:
@DavidHerreros I don't think this is the right place for this. BoxerCommand is a bit of an exception but in general, this piece of code should go inside the protocol itself.